### PR TITLE
Add config flow to philips_js

### PR DESCRIPTION
**Pull request by elupus**
## Proposed change

Adjust documentation for config flow change

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/45784

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/philips_js.markdown [hunk 1]
```diff
@@ -8,11 +8,14 @@ ha_release: 0.34
...
 ha_domain: philips_js
+ha_config_flow: true
 ---
 
 The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API. Instructions on how to activate the API and if your model is supported can be found [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices won't have JSON-interface running on port 1925. This is true at least for some models before year 2011.
 
-To add your TV to your installation, add the following to your `configuration.yaml` file:
+The preferred way to enable the integration is via **Configuration** -> **Integrations**.
+
+Alternatively a deprecated way to add your TV to your installationis via `configuration.yaml` file:
```
Comment (frenck) [inline source/_integrations/philips_js.markdown]: When an integration moves to a configuration flow, YAML configuration should be deprecated and thus removed from the documentation.

**Review by frenck — CHANGES_REQUESTED** [review]

**Comment by elupus** [comment]
I will update the documentation more linked to https://github.com/home-assistant/core/pull/46422 since it adds several new features and known issues.

**Comment by frenck** [comment]
That is a different PR and needs its own docs PR, as there is no guarantee that PR is merged before the beta cut off. Please adjust this PR first.

**Comment by elupus** [comment]
I've already done so. It just ends up rather sparse on info.

**Comment by frenck** [comment]
LOL, I missed that, sorry (its getting late here, I need to get some sleep).

**Review by frenck — APPROVED** [review]
Thanks, @elupus 👍

**Pull request merged**